pkg/dispatcher/repository_db.go

```diff
@@ -36,7 +36,7 @@ func (r Repository) Create(ctx context.Context, value db_repo.ModelBased) error
 
 	for _, err := range errs {
 		if err != nil {
-			r.logger.Error(err, "error on "+db_repo.Create+" for event "+eventName)
+			r.logger.Error("error on "+db_repo.Create+" for event : %w"+eventName, err)
```

Comment:
This file seems to mix format strings and string concatenation

Reply:
fixed